src/hooks/useTotalOutstandingTokens.ts

```diff
@@ -1,47 +1,48 @@
-import {
```

Comment:
i just commented out this whole file to avoid type errors. should be deprecated in favor of a contract call. see PR details